src/components/AssetHeader/AssetChart.tsx

```diff
@@ -83,9 +83,11 @@ export const AssetChart = ({ accountId, assetId, isLoaded }: AssetChartProps) =>
     selectTotalCryptoBalanceWithDelegations(state, filter),
   )
 
-  const delegationBalance = useAppSelector(state =>
-    selectTotalStakingDelegationCryptoByFilter(state, filter),
-  )
+  const earnBalances = useEarnBalances()
+  const delegationBalance = useMemo(() => {
+    const assetEarnBalance = earnBalances.opportunities.find(balance => balance.assetId === assetId)
+    return assetEarnBalance?.cryptoAmount ?? '0'
```

Comment:
would bnOrZero().toString() work here?

Reply:
@elmutt def more overhead than native nullish coalescing for not a lot of benefit?